en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md

```diff
@@ -104,7 +117,65 @@ Follow the steps below to enable statistics for the **endpoint** artifacts:
 1.  Select the required endpoint artifacts from the project explorer. 
 2.  Select **Statistics Enabled** and (if required) **Trace Enabled** as shown below.
      ![endpoint properties](../../assets/img/ei-analytics/endpoint-properties.png)
+     
+### Enabling load balancing Data Agent for Publishing to Analytics
+
+You can send events to multiple Analytics servers either by sending the same event to many Analytics servers or by load balancing events among a set of servers. This handles the fail-over problem. When events are load balanced within a set of servers and if one receiver cannot be reached, events are automatically sent to the other available and active Analytics servers. The following scenarios are covered in this section.
+
+#### Load balancing across group of servers 
+
+For this functionality, include the server URL in the Data Agent as a general Analytics receiver URL. The URL should be entered in a comma separated format as shown below.
+
+eg: `tcp://<Analytics-1>:<port>,tcp://<Analytics-2>:<port>,tcp://<Analytics-3>:<port>`
+
+sample: `tcp://10.100.2.32:7611, tcp://10.100.2.33:7611, tcp://10.100.2.34:7611`
+
+![lb events to servers](../../assets/img/ei-analytics/ob-lb-events-to-servers.png)
+    
+This handles failover as follows: If Analytics Receiver-1 is marked as down, then the Data Agent will send the data only to the Analytics Receiver-2 and Analytics Receiver-3 in a round robin manner. When the Analytics Receiver-1 becomes active after some time, the Data Agent automatically detects it, adds it to the operation, and again starts to load balance between all three receivers. This functionality significantly reduces the loss of data and provides more concurrency.
+
+#### Load balancing across multiple groups of servers  
+
+In this setup, there are two sets of servers that are referred to as set-A and set-B. 
+You can send events to both the sets. You can also carry out load balancing for both sets as mentioned in load balancing between a set of servers. 
```

Comment:
"....as mentioned in load balancing between a set of servers. "
where is this mentioned?
@ramindu90

Reply:
@nilminiwso2 this has to be corrected as "as mentioned in Load balancing across group of servers "